.github/label.sh

```diff
@@ -28,7 +28,7 @@ LABELS=$LABELS$(label_match reg:benchmarking '^benchmark/')
 LABELS=$LABELS$(label_match reg:example '^examples/')
 LABELS=$LABELS$(label_match reg:build '(cm|CM)ake')
 LABELS=$LABELS$(label_match reg:ci-cd '(^\.github/|\.yml$)')
-LABELS=$LABELS$(label_match reg:documentation '^doc/')
+LABELS=$LABELS$(label_match reg:documentation '^doc/|\.md$')
```

Comment:
Do you think we should also include `*.in` and `*.hpp*`, for the files in `headers` and `examples`?

Reply:
I would not consider hpp files to be documentation. We generate documentation from them but they're not documentation only. Some hpp files can be sitting in private Ginkgo space as well where documentation policies can be more lax.